backend/api/Areas/Admin/Controllers/UserController.cs

```diff
@@ -53,18 +55,27 @@ public UserController(ILogger<UserController> logger, IConfiguration configurati
         /// </summary>
         /// <param name="page"></param>
         /// <param name="quantity"></param>
-        /// <param name="sort"></param>
         /// <returns>Paged object with an array of users.</returns>
         [HttpGet("/api/admin/users")]
-        public IActionResult GetUsers(int page = 1, int quantity = 10, string sort = null) // TODO: sort and filter.
+        public IActionResult GetUsers()
         {
-            if (page < 1) page = 1;
-            if (quantity < 1) quantity = 1;
-            if (quantity > 50) quantity = 50;
+            var uri = new Uri(this.Request.GetDisplayUrl());
+            var query = Microsoft.AspNetCore.WebUtilities.QueryHelpers.ParseQuery(uri.Query);
+            _logger.LogDebug(query.ToString());
+            return GetUsers(new EModel.UserFilter(query));
+        }
 
-            var results = _pimsAdminService.User.GetNoTracking(page, quantity, sort);
+        /// <summary>
+        /// GET - Returns a paged array of users from the datasource.
```

Comment:
POST?

Reply:
Gives us two ways to make a filtered request for users.  We can pass them as query string parameters in the GET or in a JSON body in a POST.

Reply:
I just meant the comment seemed wrong.